pkg/build/debian.go

```diff
@@ -27,11 +27,16 @@ func Debian(manifest model.Manifest) error {
 	if err := util.RunMake(manifest, "istio", nil, "sidecar.deb"); err != nil {
 		return fmt.Errorf("failed to build sidecar.deb: %v", err)
 	}
-	if err := util.CopyFile(path.Join(manifest.GoOutDir(), "istio-sidecar.deb"), path.Join(manifest.OutDir(), "deb", "istio-sidecar.deb")); err != nil {
-		return fmt.Errorf("failed to package istio-sidecar.deb: %v", err)
+	if util.FileExists(path.Join(manifest.RepoOutDir("istio"), "deb")) {
+		if err := util.CopyFile(path.Join(manifest.RepoOutDir("istio"), "istio-sidecar.deb"), path.Join(manifest.OutDir(), "deb", "istio-sidecar.deb")); err != nil {
+			return fmt.Errorf("failed to package istio-sidecar.deb: %v", err)
+		}
 	}
 	if err := util.CreateSha(path.Join(manifest.OutDir(), "deb", "istio-sidecar.deb")); err != nil {
 		return fmt.Errorf("failed to package istio-sidecar.deb: %v", err)
 	}
+	if err := util.CopyFilesToDir(path.Join(manifest.GoOutDir(), "deb"), path.Join(manifest.OutDir(), "deb")); err != nil {
+		return fmt.Errorf("failed to package istio-sidecar.deb: %v", err)
+	}
```

Comment:
why do we have this? isn't the above change sufficient?

Reply:
8am? :)

Reply:
I put this  CopyFilesToDir in the wrong place. This is simply needed to make the dryrun check pass and not block the gate. We can merge without it if you like. Please advise.